### PR TITLE
Update yahs to 1.2a.2

### PR DESCRIPTION
**Pull request by BiocondaBot**
Update [`yahs`](https://bioconda.github.io/recipes/yahs/README.html): **1.2a.1** &rarr; **1.2a.2**

[install with bioconda](http://bioconda.github.io/recipes/yahs/README.html) [Conda](https://anaconda.org/bioconda/yahs/files)

Info | Link or Description
-----|--------------------
Recipe | [`recipes/yahs`](https://github.com//bioconda/bioconda-recipes/tree/bump/yahs/recipes/yahs) (click to view/edit other files)
Summary | YaHS, yet another Hi-C scaffolding tool.
Home | [https://github.com/c-zhou/yahs](https://github.com/c-zhou/yahs)
Releases |[https://github.com/c-zhou/yahs/tags](https://github.com/c-zhou/yahs/tags)
Author | `@c-zhou`
***

This pull request was automatically generated (see [docs](https://bioconda.github.io/contributor/updating.html)).

### CODE REVIEW

**Review by ksenia-krasheninnikova — APPROVED** [review]

**Review by ksenia-krasheninnikova — APPROVED** [review]

**Review by c-zhou — APPROVED** [review]

**Review by c-zhou — COMMENTED** [review]
it would be good to also include the tool `juicer` in the recipe.

**Comment by bgruening** [comment]
@c-zhou does this look good to you?

**Review by c-zhou — COMMENTED** [review]
Thanks Björn. That looks good. For consistency, you could alternatively put `juicer -h` in the test section instead of including it in `run`. The was a problem with `juicer` exit code. But it has now been fixed.

**Review by c-zhou — COMMENTED** [review]
I should also have mentioned `juicer` will give an exit code 1 while `juicer -h` will give an exit code 0. I am not sure if this will affect the building as you have `juicer` command included in the `run` section.

**Comment by bgruening** [comment]
Oo, no juicer conda package. How important is that? It its not easy to get to run :(

File: recipes/yahs/meta.yaml [hunk 1]
```diff
@@ -19,11 +19,14 @@ requirements:
   host:
     - make
     - zlib
+  run:
+    - juicer
```
Comment (c-zhou) [inline recipes/yahs/meta.yaml:23]: Oh, by `juicer`, I do not mean the juicer_tools from the Aiden lab. It is a tool that comes with `yahs` and `agp_to_fasta` (https://github.com/c-zhou/yahs#other-tools). We might need to delete these two lines? Apologies for the misleading.

**Review by c-zhou — APPROVED** [review]

**Review by bgruening — APPROVED** [review]

**Comment by bgruening** [comment]
@BiocondaBot please merge

**Pull request merged**